README.md

```diff
@@ -1 +1,54 @@
-# azure-fhir-service
\ No newline at end of file
+# azure-fhir-service
```

Comment:
we'll want to update this README.md. Fastest path would probably be

- make sure you have the latest CLI
- `mass bundle new` 
- name it `azure-fjhir-service`
- description `Azure FHIR Service enables rapid exchange of health data using the Fast Healthcare Interoperability Resources (FHIR) data standard.`

That will generate the whole bundle but copy-paste that README.md to this repo.